### ISSUE TITLE
Fix Damage Tilt

### ISSUE THREAD

**Issue by PlasmaFox256** [BTW-Community/BTW-Public#247]
Post 1.2.5, the screen no longer tilts towards the damage source like it used to. I'm pretty sure the line of code still exists but Mojang just screwed something up in an update and never fixed it since.

**Comment by Hirachosan** [comment]
yes, and iirc there's even videos on youtube that explain where this error is and how to fix it. Came across some before I learned to code.. Never checked if FC ever fixed it. Its not really noticeable for me personally. If it's not fixed we probably should look it up and have it fixed easlily

**Comment by ammoore00** [comment]
Yeah IIRC the issue is just a single number not being synced from server to client and is an easy fix

**Comment by ammoore00** [comment]
Idk about label, didn't feel right to call it a bug so :woman_shrugging:

**Comment by EmreKumas** [comment]
I've been testing this for a couple hours, and this bug has already been fixed by @Arminias in this commit: https://github.com/BTW-Community/BTW-source/commit/c3d8842e94bacf2672b564ebe07b7e435e789027

I've tested with:
- Minecraft 1.2.5 version, which has directional damage
- Minecraft 1.14 version (any version above 1.2.5), which does not have directional damage

And the behaviour in BTW 3.0 matches exactly to the 1.2.5 version. So this bug can be closed.

**Comment by Hirachosan** [comment]
nice :) thanks for clearing that up, will close this then
